plugins/attachment-resources/src/components/AttachmentRefInput.svelte

```diff
@@ -183,26 +183,31 @@
     await tick()
     const list = inputFile.files
     if (list === null || list.length === 0) return
+    const promises: Promise<any>[] = []
```

Comment:
Let's use our RateLimiter class and some limit like 10 at a time. thanks

Reply:
Thanks, fixed.